api/v1beta1/agentserviceconfig_types.go

```diff
@@ -50,6 +50,13 @@ type MustGatherImage struct {
 	Url string `json:"url"`
 }
 
+type ImageServiceOptions struct {
```

Comment:
Maybe `ImageServiceConfig`? Naming is hard.
Also I like splitting this out into a separate struct.